src/Main.java

```diff
@@ -1,5 +1,93 @@
+import java.util.Random;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+//        // task 1
+        System.out.println(findSymbolOccurance(null, 'a'));
+//        // task 2
+        System.out.println(findWordPosition("Apollo", "pollo"));
+        System.out.println(findWordPosition("Apple", "Planet"));
+//        // task 3
+        System.out.println(stringReverse("Hello"));
+        System.out.println(stringReverse2("Java"));
+//        // task 4
+        System.out.println(isPalindrome("ERE"));
+        System.out.println(isPalindrome("Java"));
+        // task 5
+        System.out.println(wordGame());
+
+    }
+    public static int findSymbolOccurance(String word, char letter) {
+        int counter = 0;
+        if (word == null) {
+            throw new IllegalArgumentException("Word cannot be null.");
+        }
+        for (int i = 0; word.length() > i; i++) {
+
+            if (word.charAt(i) == letter) {
+                counter++;
+            }
+        }
+        return counter;
+    }
+
+    public static int findWordPosition(String source, String target) {
+        if (source == null || target == null) {
+            throw new IllegalArgumentException("Word cannot be null.");
```

Comment:
You're trying to validate 2 arguments, `source` and `target`. But your message says `Word cannot be null.` but there is no such parameter here. Please do not copy your messages, write them always by hand.
So as a user, I cannot understand what I did wrong, is something wrong with `source` or with `target` string? Please think of it and change your error message.